docs/topics/mta-web-manage-columns.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: CONCEPT
+[id="manage-columns_{context}"]
+= Managing displayed columns
+
+When viewing applications in the *Application Inventory* window, you can set which information columns appear in the list of applications
+
+.Procedure
+
+. In *Migration* view, click *Application inventory*.
+. Click *Manage Collumns* to open a dialog that lists all the available columns.
++
+.Manage columns
+image::mta-web-manage-columns.png[Manage columns]
+
+. Check the box to the left of each columns that you want to appear in the application inventory.
+. Click *Save* to set your selection.
```

Comment:
This is not implemented in MTA 7.0.0

Reply:
@abrugaro - are you ok if we comment out the `include` statement in the `master.adoc`

which would remove this entire page

Reply:
yes please keep this removed.

Reply:
@anarnold97 sounds good to me, thanks!

Reply:
Link to page commented out of master.adoc file